### ISSUE TITLE
need to update for julia 0.5

### ISSUE THREAD

**Issue by CarloLucibello** [JuliaGraphs/LightGraphsExtras.jl#13]
I'm taking care of this

**Comment by CarloLucibello** [comment]
@jpfairbanks looks like once I checkout the "0.5-update" branch of graph matrices most problems are solved, except for A_mul_B not defined for the Nonbacktracking type defined here. Can you define take a look at this?

**Comment by jpfairbanks** [comment]
Yeah the problem is that when we define * in 0.4 we need to also define A_mul_B in 0.5.
https://github.com/JuliaGraphs/LightGraphs.jl/blob/master/src/spectral.jl#L239 is the relevant definition.
```julia
function A_mul_B!(C, nbt::Nonbacktracking, B)
    # computs C = A*B
    for i in 1:size(B,2)
        C[:,i] = A*B[:,i]
    end
    return C
end
```

**Comment by CarloLucibello** [comment]
@sbromberger do we have any schedule for the next LG tag? can't release a LGExtras 0.5 compatible version until that

**Comment by sbromberger** [comment]
I was waiting for @jpfairbanks to finish the LinAlg submodule. If that looks like it's going to be delayed past this weekend I will tag a new version on Sunday.

**Comment by sbromberger** [comment]
Closing this out since we've merged #20 and a new tag is in progress.
